modules/_labs/pagination/react/lib/Pages.tsx

```diff
@@ -0,0 +1,124 @@
+import {css, jsx} from '@emotion/core';
+import {IconButton} from '@workday/canvas-kit-react-button';
+import canvas from '@workday/canvas-kit-react-core';
+import _ from 'lodash';
+import React, {useRef} from 'react';
+
+import {PaginationAriaLabelsDefault} from './PaginationAriaLabels';
+
+/** @jsx jsx */
```

Comment:
This might have to be moved to the top of the file to avoid the Typescript error that's preventing Chromatic and builds from running

Reply:
done